types/prettier/index.d.ts

```diff
@@ -280,9 +294,10 @@ export interface SupportOption {
     redirect?: SupportOptionRedirect;
     description: string;
     oppositeDescription?: string;
-    default: SupportOptionValue;
+    default: SupportOptionValue | SupportOptionDefault[];
     range?: SupportOptionRange;
     choices?: SupportOptionChoice[];
+    category: SupportOptionCategory;
```

Comment:
`category` is supposed to be a string.

Reply:
Right now it's "JavaScript", but most likely the developers want to add more categories, so I added an `enum` (see [line 280](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37003/files#diff-cc5ba5a18b4122d24149ba472e7720c3R280))

Reply:
I meant it could be any string, and there is no `prettier.SupportOptionCategory`. `enum` not only declares a variable in the type-space but also in the value-space.

```suggestion
    category: string;
```